unitTests/ui/DynamicFacet/DynamicFacetBreadcrumbsTest.ts

```diff
@@ -56,6 +56,9 @@ export function DynamicFacetBreadcrumbsTest() {
       initializeComponent();
 
       const [breadcrumb] = valueElements();
+      $$(breadcrumb)
```

Comment:
Without removing the svg, `breadcrumb.childNodes[0].nodeValue` should target the text correctly

Reply:
is the text always guaranteed to be the `childNodes[0]` ? That's what I was thinking when I wrote this

Reply:
Yes except if someone changes the breadcrumb in the future, in that case, it's expected that the test fail 👍